spring-cloud-previews/google-cloud-aiplatform-spring-starter/src/main/java/com/google/cloud/aiplatform/v1/spring/JobServiceSpringAutoConfiguration.java

```diff
@@ -232,6 +232,40 @@ public JobServiceSettings jobServiceSettings(
           .cancelHyperparameterTuningJobSettings()
           .setRetrySettings(cancelHyperparameterTuningJobRetrySettings);
 
+      RetrySettings createNasJobRetrySettings =
```

Comment:
This corresponds to changes from https://github.com/googleapis/google-cloud-java/pull/9073